app/models/game.rb

```diff
@@ -6,4 +6,43 @@ class Game < ActiveRecord::Base
 
   delegate :pawns, :rooks, :knights, :bishops, :queens, :kings, to: :pieces
   scope :available, -> { where('white_player_id IS NULL OR black_player_id IS NULL') }
+
+  def populate_board!
+    #- white - 16
+    # Pawn - 8
+    (0..7).each do |i|
+      pieces.create(type: 'Pawn', x_position: i, y_position: 0, color: 0)
+    end
+    # Rook - 2
+    pieces.create(type: 'Rook', x_position: 0, y_position: 0, color: 0)
+    games.pieces.create(type: 'Rook', x_position: 7, y_position: 0, color: 0)
+    # Knight - 2
+    pieces.create(type: 'Knight', x_position: 1, y_position: 0, color: 0)
+    pieces.create(type: 'Knight', x_position: 6, y_position: 0, color: 0)
+    # Bishop - 2
+    pieces.create(type: 'Bishop', x_position: 2, y_position: 0, color: 0)
+    pieces.create(type: 'Bishop', x_position: 5, y_position: 0, color: 0)
+    # Queen
+    pieces.create(type: 'Rook', x_position: 4, y_position: 0, color: 0)
+    # King
+    pieces.create(type: 'Rook', x_position: 3, y_position: 0, color: 0)
+    #- black - 16
+    # Pawn - 8
+    (0..7).each do |i|
+      pieces.create(type: 'Pawn', x_position: i, y_position: 7, color: 1)
+    end
+    # Rook - 2
+    pieces.create(type: 'Rook', x_position: 0, y_position: 7, color: 1)
+    pieces.create(type: 'Rook', x_position: 7, y_position: 7, color: 1)
+    # Knight - 2
+    pieces.create(type: 'Knight', x_position: 1, y_position: 7, color: 1)
+    pieces.create(type: 'Knight', x_position: 6, y_position: 7, color: 1)
+    # Bishop - 2
+    pieces.create(type: 'Bishop', x_position: 2, y_position: 7, color: 1)
+    pieces.create(type: 'Bishop', x_position: 5, y_position: 7, color: 1)
+    # Queen
+    pieces.create(type: 'Rook', x_position: 3, y_position: 7, color: 1)
+    # King
+    pieces.create(type: 'Rook', x_position: 4, y_position: 7, color: 1)
+  end
```

Comment:
To test this, start by trying something like this

```Ruby
# pieces_spec.rb
describe '.populate_board!' do
  it 'creates correct # of pieces' do
    game = FactoryGirl.create(:game)
    expect(game.pieces.length).to eq 32

     expect(game.pawns.length).to eq 16
     expect(game.rooks.length).to eq 4
     # ...etc...
  end
end
```

Then once you have that test running. You check that the correct colors are set and correct coordinates.